frontend/src/pages/project/RoleDetailsBySlugPage/components/ProjectRoleModifySection.utils.tsx

```diff
@@ -137,7 +137,8 @@ export const projectRoleFormSchema = z.object({
       [ProjectPermissionSub.Tags]: GeneralPolicyActionSchema.array().default([]),
```

Comment:
(ignore file placement of comment)

There does not seem to be a "Connect" action for App Connections in the organization role page
<img width="1482" alt="image" src="https://github.com/user-attachments/assets/b954b4f0-f1ec-4612-ac64-7981b669cfbf" />